docs/getting-started/README.md

```diff
@@ -1,198 +1,203 @@
-  - Most cloud providers k8s offerings work for this purpose...
-    - but ingress does not work out of the box for GKE clusters.
-    - For now, GKE users should consider using the
-      [nginx ingress](https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke).
-- [Install Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#installing-tekton-pipelines)
-  - Pipelines is the backbone of Tekton and will allow us to accomplish the work
-    we plan to do.
-- [Install Triggers](../install.md)
-  - Of course we need to install our project as well, so we can accept and
-    process events into PipelineRuns!
-- Pick a GitHub repo with a Dockerfile as your build object (or you can fork
-  [this one](https://github.com/iancoffey/ulmaceae)).
-  - Clone this repo locally - we will come back to this repo later.
-
-## Configure the cluster
-
-Now that we have our cluster ready, we need to setup our getting-started
-namespace and RBAC. We will keep everything inside this single namespace for
-easy cleanup. In the unlikely event that you get stuck/flummoxed, the best
-course of action might be to just delete this namespace and start fresh.
-You should take note of the _ingress_ subdomain, or the external IP of
-your cluster, you will need this for your Webhook in later steps.
-
-- Create the _getting-started_ namespace, where all our resources will live.
-  - `kubectl create namespace getting-started`
-- [Create the admin user, role and rolebinding](./rbac/admin-role.yaml)
-  - `kubectl -n getting-started apply -f ./docs/getting-started/rbac/admin-role.yaml`
-- If you have a cluster secret for a Let's Encrypt cert already provisioned
-you will need to export it, and reimport it to the _getting-started_ namespace.
-The following is a general example of what you'd need to do.
-  - ```bash
-	kubectl get secret <name> --namespace=<namespace> -o yaml |\
-	   grep -v '^\s*namespace:\s' |\
-	   kubectl apply --namespace=<new namespace> -f -
-	```
-- [Create the create-webhook user, role and rolebinding](./rbac/webhook-role.yaml)
-  - `kubectl -n getting-started apply -f ./docs/getting-started/rbac/webhook-role.yaml`
-  - This will allow our webhook to create the things it needs to.
+# Tutorial: Getting started with Tekton Triggers
 
-## Install the Pipeline and Triggers
+The following tutorial walks you through building and deploying a Docker image using 
+Tekton Triggers to detect a GitHub webhook request and execute a `Pipeline`.
 
-### [Install the Pipeline](./pipeline.yaml)
+## Overview
 
-Now we have to install the Pipeline we plan to use and also our Triggers
-resources.
+In this tutorial, you will:
 
-`kubectl -n getting-started apply -f ./docs/getting-started/pipeline.yaml`
+1. Set up a [`Pipeline`](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md) that builds a Docker image using
+   [kaniko](https://github.com/GoogleContainerTools/kaniko) and deploys it locally using the `kubectl image` command. The
```

Comment:
`kubectl image` :thinking: 

I think it should be `kubectl apply` or `kubectl create`

Reply:
I copied this from here: https://github.com/tektoncd/triggers/blob/main/docs/getting-started/README.md#what-does-it-do.  It says "The Pipeline will build a Docker image with kaniko and deploy it locally via kubectl image." What should we say here instead?

Reply:
I'm all for keeping it super-simple unless there's a very specific reason we must point out it's a "kubectl Docker image." Changed it to "...and deploys it locally to your Kubernetes cluster." Let me know if that works.